### PR TITLE
36190 no debt info

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Description
If there is no debt balance this PR adds text to show their debt balance is $0 and if they have no debt this adds a link for users to click on to learn about VA debt

## Testing done
Added new unit tests and manual testing was done

## Acceptance criteria
- [x] Changes above have been made per mockups
- [x] A PR has been submitted for review

### CODE REVIEW

**Review by cohnjesse — CHANGES_REQUESTED** [review]
@tmitchellgcio can you add a description to this ticket of what it is for and some background on why the changes are needed? Please add this to all tickets moving forward as well.

**Comment by tmitchellgcio** [comment]
> @tmitchellgcio can you add a description to this ticket of what it is for and some background on why the changes are needed? Please add this to all tickets moving forward as well.

Usually I attach the ticket, which I forgot to do this time. Literally the first time I've forgotten in months, but are you saying to add a description even if it's in the ticket? The ticket should explain all of that and I've rarely needed to add descriptions to the tickets.

**Comment by cohnjesse** [comment]
@tmitchellgcio No worries, yes even if you attach the ticket please add all of the details in the template that gets created when you create a ticket ( a description,  the testing that was done, screenshots if applicable, acceptance criteria ). Having the original ticket attached is great, and you should do that, however tickets are often lacking in details and we should strive to make each PR so that an engineer can just look at the PR and get all of the information they need.

**Comment by tmitchellgcio** [comment]
@cohnjesse no problem. Updated and will do this more in the future

**Review by adamwhitlock1 — APPROVED** [review]
LGTM overall. 

There are a couple linting warning for prop validation and an inline arrow function that you may want to clean up as time permits, but not blocking.

**Comment by tmitchellgcio** [comment]
I'm going to fix some of this linting stuff before I merge this. I also can't merge without @cohnjesse for his requested changes.

**Comment by hdjustice** [comment]
Hi @cohnjesse Is there anything else we need on this one?  Thanks!

**Comment by cohnjesse** [comment]
@hdjustice @tmitchellgcio I would love to see the original ticket linked on this as well as acceptance criteria there is for the PR. I also think you can get rid of the definition of done portion in the ticket. For future reference [here](https://github.com/department-of-veterans-affairs/vets-website/pull/19538) is a good reference of the level of detail/ what to put into each PR description at a minimum.

**Comment by tmitchellgcio** [comment]
@cohnjesse The ticket is linked and has been since yesterday

**Comment by tmitchellgcio** [comment]
@cohnjesse you don't need to put the ticket in the description. You can connect your ticket with the connect issue button. That's how it's connected here.

**Comment by tmitchellgcio** [comment]
@cohnjesse completed. Anything else you need before you look at the PR?

**Review by cohnjesse — APPROVED** [review]
LGTM

File: src/applications/personalization/dashboard/tests/e2e/benefit-payments-and-debt.cypress.spec.js
Comment (tlei123) [inline src/applications/personalization/dashboard/tests/e2e/benefit-payments-and-debt.cypress.spec.js:158]: Looks like the code-comment here should say "is not shown". The subsequent assertions are 'not.exist'.

**Review by tlei123 — APPROVED** [review]
Other than the issues flagged by the linter and me [for the code-comments], LGTM.

File: src/applications/personalization/dashboard/tests/e2e/benefit-payments-and-debt.cypress.spec.js
Comment (tmitchellgcio) [reply src/applications/personalization/dashboard/tests/e2e/benefit-payments-and-debt.cypress.spec.js:181]: I'd rather it be done right. I will change that sorry I should have paid attention there

**Pull request merged**
